File: AppSets-Android/appsets/src/main/kotlin/xcj/app/appsets/im/RabbitMQBrokerConfig.kt
package xcj.app.appsets.im

class RabbitMQBrokerConfig(
    private val mRabbitMQBrokerProperty: RabbitMQBrokerProperty
) : MessageBrokerConfig {

    fun getRabbitProperty(): RabbitMQBrokerProperty {
        return mRabbitMQBrokerProperty
    }

}

File: AppSets-iOS/AppSets/com/app/appsets/im/RabbitMQBrokerConfig.swift
//
//  RabbitMQBrokerConfig.swift
//  AppSets
//
//  Created by caiju Xu on 2024/7/9.
//

import Foundation

struct RabbitMQBrokerConfig : MessageBrokerConfig {
    
    let property:RabbitMqBrokerProperty
    
    init(_ property: RabbitMqBrokerProperty) {
        self.property = property
    }
}
